google/cloud/bigtable/data/_metrics/handlers/opentelemetry.py

```diff
@@ -211,7 +212,8 @@ def on_attempt_complete(
             combined_throttling += op.flow_throttling_time_ms
         self.otel.throttling_latencies.record(combined_throttling, labels)
         self.otel.application_latencies.record(
-            attempt.application_blocking_time_ms + attempt.backoff_before_attempt_ms, labels
+            attempt.application_blocking_time_ms + attempt.backoff_before_attempt_ms,
+            labels,
         )
```

Comment:
Maybe this was added in an earlier PR, but first response latencies is actually per operation not per attempt.